src/main/resources/openshift/deploy.sh

```diff
@@ -4,6 +4,10 @@ set -e
 
 OCP_PROJECT=rhamt
 
+if [ -z "$RHAMT_VOLUME_CAPACITY" ]; then
+    RHAMT_VOLUME_CAPACITY=10Gi
```

Comment:
Do we have some recommendation in documentation what is good to set up? I mean minimum and maximum values for a typical sized applications?